Library/Homebrew/extend/git_path.rb

```diff
@@ -7,12 +7,14 @@
 # Extensions to {Pathname} for querying Git repository information.
 # @see Utils::Git
 # @api private
-module GitRepositoryExtension
+class GitPath < SimpleDelegator
```

Comment:
```suggestion
class GitRepository < SimpleDelegator
```
Maybe? `GitRepoPath` is fine too.